spec/lcm/integration/shared_contexts_for_lcm.rb

```diff
@@ -75,19 +77,19 @@ def create_workspace_csv(workspaces, client_id_column)
 shared_context 'lcm bricks' do
```

Comment:
Similar code found in shared_contexts_for_lcm.rb:57, shared_contexts_for_lcm.rb:67 (mass = 78)